utils/index.js

```diff
@@ -297,6 +319,26 @@ const currencyFilter = (number, locale = 'en') => {
   return `$${filteredAmount}`
 }
 
+/*
+ * Accepts an ISO-format date (1999-09-30)
+ * and returns a string formatted "dd mm yyyy" (30 09 1999)
+ *
+ * @param {*} date a string ISO-format date
+ */
+const isoDateHintText = date => {
+  if (!validator.isISO8601(date)) {
+    throw new Error(`[GET /login/dateOfBirth] Bad date "${date}": must be a valid ISO date`)
+  }
+
+  const dateParts = date.split('-')
+
+  if (dateParts.length !== 3 || dateParts[2].length > 2) {
+    throw new Error(`[GET /login/dateOfBirth] Bad date "${date}": must be formatted yyyy-mm-dd`)
+  }
+
+  return `${dateParts[2]} ${dateParts[1]} ${dateParts[0]}`
+}
+
```

Comment:
I moved this method up here from lower down since it's a pug filter. Didn't change the code.